integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableRenameCommand.scala

```diff
@@ -93,6 +96,22 @@ private[sql] case class CarbonAlterTableRenameCommand(
       timeStamp = System.currentTimeMillis()
       schemaEvolutionEntry.setTime_stamp(timeStamp)
       renameBadRecords(oldTableName, newTableName, oldDatabaseName)
+
```

Comment:
Why the order changed?

Reply:
if hive and carbon both the tables  points to the same location, and we do folder rename first then then hive alter command will fail.